src/sentry/integrations/github/client.py

```diff
@@ -276,13 +291,22 @@ def _populate_trees(self, repositories: List[Dict[str, str]]) -> Dict[str, RepoT
                     repo_info, only_use_cache, (3600 * 24) + (3600 * (index % 24))
                 )
             except ApiError as error:
-                self._populate_trees_process_error(error, extra)
+                should_count_error = self._populate_trees_process_error(error, extra)
+                if should_count_error:
```

Comment:
Yeah, it's a little verbose, but it's otherwise not obvious from the name of `_populate_trees_process_error` what it's going to return, and therefore not clear why it should be the condition. (I'm generally always going for the "don't make them think" rule as much as is reasonable.)

Reply:
Sure